modules/akamaiDapRtdProvider.js

```diff
@@ -53,60 +61,95 @@ export function addRealTimeData(rtd) {
  * Real-time data retrieval from Audigent
  * @param {Object} reqBidsConfigObj
  * @param {function} onDone
- * @param {Object} rtdConfi
+ * @param {Object} rtdConfig
  * @param {Object} userConsent
  */
 export function getRealTimeData(bidConfig, onDone, rtdConfig, userConsent) {
+  let entropyDict = JSON.parse(localStorage.getItem(DAP_CLIENT_ENTROPY));
+  let loadScriptPromise = new Promise((resolve, reject) => {
+    if (rtdConfig && rtdConfig.params && rtdConfig.params.dapFpTimeout && Number.isInteger(rtdConfig.params.dapFpTimeout)) {
+      setTimeout(reject, rtdConfig.params.dapFpTimeout, Error('DapFP script could not be loaded'));
+    }
+    if (entropyDict && entropyDict.expires_at > Math.round(Date.now() / 1000.0)) {
+      logMessage('Using cached entropy');
+      resolve();
+    } else {
+      if (typeof window.dapCalculateEntropy === 'function') {
```

Comment:
Is this ability to define the entropy function in advance in the doc?

Reply:
Yes, in the cases where the script is already loaded and the function already exists, then skip the downloading and just call the function directly.

Reply:
please document the two options in https://github.com/prebid/prebid.github.io/pull/3700/files